docs/fr-FR/newbs.md

```diff
@@ -0,0 +1,23 @@
+# Le Guide pour débutant complet à QMK
+
+QMK est un firmware Open Source pour votre clavier mécanique. Vous pouvez utiliser QMK pour customiser votre clavier de manière simple et puissante. Tout le monde, du débutant complet au développeur avancé, ont utilisé avec succès QMK pour customiser leur clavier. Ce guide vous aidera à faire de même, quelles que soient vos compétences.
+
+Vous voulez savoir si votre clavier peut utiliser QML? Si c'est un clavier mécanique que vous avez vous-même construit, il y a de bonnes chances que vous pouvez. Nous supportos un [grand nombre de "hobbyist boards"](http://qmk.fr/keyboards), donc même si votre clavier ne peut pas utiliser QMK, vous ne devriez pas avoir trop de problème pour en trouver un qui vous convienne.
+
+## Vue d'ensemble
+
+Il y a 7 sections principales dans ce guide:
+
+* [Pour débuter](fr-FR/newbs_getting_started.md)
+* [Compiler votre premier firmware en utilisant la ligne de commande](fr-FR/newbs_building_firmware.md)
+* [Compiler votre premier firmware en utilisant l'interface graphique en ligne](fr-FR/newbs_building_firmware_configurator.md)
+* [Téléverser le Firmware](fr-FR/newbs_flashing.md)
+* [Test et Débuggage](fr-FR/newbs_testing_debugging.md)
+* [Bonnes pratiques Git](fr-FR/newbs_best_practices.md)
+* [Ressources d'apprentissage](fr-FR/newbs_learn_more_resources.md)
+
+Ce guide se concentre sur aider quelqu'un qui n'a jamais compilé de logiciel avant. Il fait des choix et des recommandations basées sur ce point de vue. Il y a des méthodes alternatives pour beaucoup de ces procédures, et nous supportons la plupart de ces alternatives. Si vous avez un doute sur comment accomplir une tâche, vous pouvez [nous demander de l'aide](fr-FR/getting_started_getting_help.md).
```

Comment:
`Il fait des choix et des recommandations basées sur ce point de vue.` This is weird

Reply:
Reworked this phrase and the previous which was indeed weird